.eslintrc.js

```diff
@@ -317,6 +317,7 @@ module.exports = {
         '**/storybook/*.js',
       ],
       'rules': {
+        'require-jsdoc': 0,
```

Comment:
This would exempt all the test files and init_tests etc. from JSDoc; could add a separate exeption block for storybook specifically, and leave the JSDoc on test files?